CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ option(BUILD_SHARED_LIBS "Build shared avif library" ON)
 
 option(AVIF_ENABLE_WERROR "Treat all compiler warnings as errors" OFF)
 
-option(AVIF_ENABLE_YCGCO_R "Enable experimental YCgCo-R matrix code for encoding" OFF)
+option(AVIF_ENABLE_YCGCO_R_ENCODING "Enable experimental YCgCo-R matrix code for encoding" OFF)
```

Comment:
Why does the use of YCgCo-R matrix for encoding need to stay experimental?

Reply:
Fixed the comment.